build.gradle

```diff
@@ -14,9 +14,9 @@ subprojects {
     kotlin {
       ktlint().editorConfigOverride([
         'ktlint_standard_filename': 'disabled',
+        'ktlint_standard_property-naming': 'disabled',
```

Comment:
Despite disagreeing with a bunch of these, it's probably best to comply with it rather than disable it.

Reply:
375b62c1a3dd46f6a5a27d00831d731d1f7c55b3